Update cors.md (#30015)
dotnet/AspNetCore.Docs: aspnetcore/security/cors.md

By [Rick Anderson](https://twitter.com/RickAndMSFT) and [Kirk Larkin](https://twitter.com/serpent5)

> This article shows how **C**ross-**O**rigin **R**esource **S**haring (https://developer.mozilla.org/docs/Web/HTTP/CORS)) in enabled in an ASP.NET Core app.

Author: ```suggestion
This article shows how **C**ross-**O**rigin **R**esource **S**haring ([CORS](https://developer.mozilla.org/docs/Glossary/CORS)) in enabled in an ASP.NET Core app.
```

> This article shows how **C**ross-**O**rigin **R**esource **S**haring ([CORS](https://developer.mozilla.org/docs/Glossary/CORS)) in enabled in an ASP.NET Core app.

Author: ```suggestion
This article shows how **C**ross-**O**rigin **R**esource **S**haring (https://developer.mozilla.org/docs/Web/HTTP/CORS)) in enabled in an ASP.NET Core app.
```

> This article shows how **C**ross-**O**rigin **R**esource **S**haring ([CORS]([url]This article shows how **C**ross-**O**rigin **R**esource **S**haring ([CORS]([url](https://developer.mozilla.org/docs/Glossary/CORS))) is enabled in an ASP.NET Core app.

Author: ```suggestion
This article shows how **C**ross-**O**rigin **R**esource **S**haring ([CORS](https://developer.mozilla.org/docs/Glossary/CORS)) in enabled in an ASP.NET Core app.
```

By [Rick Anderson](https://twitter.com/RickAndMSFT) and [Kirk Larkin](https://twitter.com/serpent5)

> This article shows how **C**ross-**O**rigin **R**esource **S**haring ([CORS]([url](https://developer.mozilla.org/docs/Glossary/CORS)) in enabled in an ASP.NET Core app.

Author: ```suggestion
This article shows how **C**ross-**O**rigin **R**esource **S**haring ([CORS]([url]This article shows how **C**ross-**O**rigin **R**esource **S**haring ([CORS]([url](https://developer.mozilla.org/docs/Glossary/CORS))) is enabled in an ASP.NET Core app.
```

> This article shows how Cross-Origin Resource Sharing ([CORS]([url](https://developer.mozilla.org/docs/Glossary/CORS)) in enabled in an ASP.NET Core app.

Author: ```suggestion
```
